scripts/terraform-docs.sh

```
@@ -1,9 +1,34 @@
+  curl -sSLo ./terraform-docs.tar.gz.sha256sum https://terraform-docs.io/dl/${TERRAFORM_DOCS_VERSION}/terraform-docs-${TERRAFORM_DOCS_VERSION}.sha256sum
+  if sha256sum --ignore-missing -c "terraform-docs.tar.gz.sha256sum"; then
+    echo "✓ File verification successful"
+  else
+    echo "✗ File verification failed" >&2
+    exit 1
+  fi
+  tar -xzf terraform-docs-${TERRAFORM_DOCS_VERSION}-${PLATFORM}-amd64.tar.gz terraform-docs
+  chmod +x terraform-docs
+  echo "sudo required for moving terraform-docs to /usr/local/bin"
+  sudo mv terraform-docs /usr/local/bin/terraform-docs
+  rm terraform-docs-${TERRAFORM_DOCS_VERSION}-${PLATFORM}-amd64.tar.gz
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)